From f9a64ce561b122368118149cb24c20bf3e0c9e1c Mon Sep 17 00:00:00 2001
From: Chih-Wei Huang <cwhuang@linux.org.tw>
Date: Tue, 26 Feb 2019 14:57:00 +0800
Subject: [PATCH 08/63] Remove an unused function

This gets rid of an error from Android 9 clang:

external/efivar/src/linux.c:256:1: error: unused function 'supports_iface' [-Werror,-Wunused-function]
supports_iface(struct dev_probe *probe, enum interface_type iftype)
^
1 error generated.

Signed-off-by: Chih-Wei Huang <cwhuang@linux.org.tw>
---
 src/linux.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/src/linux.c b/src/linux.c
index 6d405af8a76..4bb453be834 100644
--- a/src/linux.c
+++ b/src/linux.c
@@ -252,15 +252,6 @@ static struct dev_probe *dev_probes[] = {
         NULL
 };
 
-static inline bool
-supports_iface(struct dev_probe *probe, enum interface_type iftype)
-{
-        for (unsigned int i = 0; probe->iftypes[i] != unknown; i++)
-                if (probe->iftypes[i] == iftype)
-                        return true;
-        return false;
-}
-
 void HIDDEN
 device_free(struct device *dev)
 {
-- 
2.26.2

